src/server/utilities/cspHeader/index.js

```diff
@@ -270,6 +272,7 @@ const directives = {
       'https://news.files.bbci.co.uk', // STY include styles
       'https://platform.twitter.com', // Social Embeds
       'https://ton.twimg.com', // Social Embeds
+      'https://news.test.files.bbci.co.uk', // STY includes
```

Comment:
On L272, we have a directive that covers the non-test domain `https://news.files.bbci.co.uk`. Rather than add another directive for an origin we control, would it make sense to use a wildcard, so `https://news.files.bbci.co.uk` becomes `https://*.files.bbci.co.uk`?

Reply:
yeah, I saw that. I saw an instance in this file where there was`news.files.bbci.co.uk` and `b.files.bbci.co.uk` which gave me the impression that we might want to be explicit in the origins declared in this file. would like to know what others think of this also

Reply:
Given a specific prefix has been used elsewhere, yeah I'd be specific in this case. I think if we get anymore variants of `*.files.bbci.co.uk` we should investigate using a wildcard.